lib/code_corps/map_utils.ex

```diff
@@ -8,18 +8,18 @@ defmodule CodeCorps.MapUtils do
   def keys_to_string(map), do: stringify_keys(map)
 
   # Goes through a list and stringifies keys of any map member
-  def stringify_keys(nil), do: nil
-  def stringify_keys(%DateTime{} = val), do: val
-  def stringify_keys(map = %{}) do
+  defp stringify_keys(nil), do: nil
+  defp stringify_keys(%DateTime{} = val), do: val
```

Comment:
@begedin the bigger point is that we should not ever (I think) be hitting a `%DateTime{}` – moreover, in both cases these functions simply return the value. They are the same function, strictly speaking.